code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -209,3 +277,75 @@
 	coat["dominia great coat, black"] = /obj/item/clothing/suit/storage/toggle/dominia/black
 	coat["dominia great coat, alternative black"] = /obj/item/clothing/suit/storage/toggle/dominia/black/alt
 	gear_tweaks += new/datum/gear_tweak/path(coat)
+
+/datum/gear/suit/mil
```

Comment:
Maybe create a military jacket selection or add them to the regular jacket selection?